default.json

```diff
@@ -263,6 +263,16 @@
     }
   ],
   "packageRules": [
+    {
+      "description": "Disable digest updates (commit-based versions) for Go modules",
+      "matchManagers": [
+        "gomod"
+      ],
+      "matchUpdateTypes": [
+        "digest"
+      ],
+      "enabled": false
+    },
```

Comment:
Now there are two rules that do the same thing. Let's remove this one and leave the one that is closest to where `digest` packages are configured.
```suggestion
```